app/controllers/idv/doc_auth_controller.rb

```diff
@@ -50,13 +50,15 @@ def flow_session
       user_session['idv/doc_auth']
     end
 
-    def add_unsafe_eval_to_capture_steps
+    def override_document_capture_step_csp
       return unless params[:step] == 'document_capture'
 
-      # required to run wasm until wasm-eval is available
       SecureHeaders.append_content_security_policy_directives(
         request,
+        # required to run wasm until wasm-eval is available
         script_src: ['\'unsafe-eval\''],
+        # required for retrieving image dimensions from uploaded images
+        img_src: ['blob:'],
       )
```

Comment:
would love to share this method between the controllers, maybe we could move it to a mixin?

Reply:
> would love to share this method between the controllers, maybe we could move it to a mixin?

Yeah, seems like this would be good, since it's pretty easy to overlook the fact that `CaptureDocController` (the hybrid flow controller) even exists 😬

Reply:
> would love to share this method between the controllers, maybe we could move it to a mixin?

Updated in d51d7944e36f6606004f49ceac37d4e87c2a0b76. Not 100% confident that I'm following conventions here (particularly with testing), so any feedback is welcome.